daemons/execd/pacemaker-execd.c

```diff
@@ -514,7 +514,7 @@ main(int argc, char **argv, char **envp)
 
 #ifdef PCMK__COMPILE_REMOTE
     if (options.port != NULL) {
-        setenv("PCMK_remote_port", options.port, 1);
+        pcmk__set_env_option(PCMK__ENV_REMOTE_PORT, options.port);
```

Comment:
I'd rather not introduce an `HA_` variant of environment variables that don't already have them. We can add an argument to `pcmk__set_env_option()` for that.

Reply:
Good point. I'll add a commit for this. My current plan is to ignore the new argument when we're *un*setting an option. The reason is that `pcmk__get_env_option()` checks for both, so it's better to make sure we fully clear it.

That's debatable though.

Reply:
Makes sense